fixup-mountpoints

```diff
@@ -27,6 +40,7 @@ case "$DEVICE" in
             -e 's block/platform/msm_sdcc.1/by-name/boot mmcblk0p33 ' \
             "$@"
         ;;
+
```

Comment:
Newlines for the rest of the devices have been added under commit of lux, and in case of reverting that would induce many conflicts